tests/nnstreamer_filter_tensorflow2_lite/unittest_filter_tensorflow2_lite.cc

```diff
@@ -367,6 +368,50 @@ TEST (nnstreamerFilterTensorFlow2Lite, manyInOutModel)
   g_free (model_file);
 }
 
+/**
+ * @brief Test for suspend mode.
+ */
+TEST (nnstreamerFilterTensorFlow2Lite, suspend)
+{
+  gchar *pipeline;
+  GstElement *gstpipe;
+  GError *err = NULL;
+  gchar *model_file;
+
+  ASSERT_TRUE (_GetModelFilePath (&model_file, 0));
+
+  /* create a nnstreamer pipeline */
+  pipeline = g_strdup_printf ("appsrc name=srcx ! application/octet-stream ! tensor_converter input-dim=3:224:224 input-type=uint8 ! tensor_filter suspend=2000 framework=tensorflow2-lite model=\"%s\" ! tensor_sink name=sink async=false",
+      model_file);
+
+  gstpipe = gst_parse_launch (pipeline, &err);
+  ASSERT_TRUE (gstpipe != nullptr);
+
+  GstElement *src_handle = gst_bin_get_by_name (GST_BIN (gstpipe), "srcx");
+  ASSERT_TRUE (src_handle != nullptr);
+  EXPECT_EQ (setPipelineStateSync (gstpipe, GST_STATE_PLAYING, UNITTEST_STATECHANGE_TIMEOUT), 0);
+
+  GstBuffer *buf = gst_buffer_new ();
+  GstMemory *mem = gst_allocator_alloc (NULL, 3 * 224 * 224, NULL);
+  gst_buffer_append_memory (buf, mem);
+
+  buf = gst_buffer_ref (buf);
+  EXPECT_EQ (gst_app_src_push_buffer (GST_APP_SRC (src_handle), buf), GST_FLOW_OK);
+
+  /** Wait for unloading the framework. */
+  g_usleep (5000000);
+
+  EXPECT_EQ (gst_app_src_push_buffer (GST_APP_SRC (src_handle), buf), GST_FLOW_OK);
+  g_usleep (1000000);
+
+  EXPECT_EQ (setPipelineStateSync (gstpipe, GST_STATE_NULL, UNITTEST_STATECHANGE_TIMEOUT), 0);
```

Comment:
When ````g_usleep(5000000);``` is performed,
Maybe the pipeline operation will unload the model.
It would be nice to check that the model has been unloaded in TC, but there is no way, right?

Reply:
Right. It is difficult to check by test that the framework is unloaded.
This is checked by log manually.